src/Entities/Collection.ts

```diff
@@ -1,6 +1,7 @@
 import { Field, ID, ObjectType } from "type-graphql"
 import { Column, Entity, Index, ObjectID, ObjectIdColumn } from "typeorm"
 import { CollectionQuery } from "./CollectionQuery"
+import { Subcollection } from "./Subcollection"
```

Comment:
I'm struggling with the naming here a bit because `Subcollection` is not really a collection but a group of collections. I know most word synonymous with `group` in our namespace are already taken. Maybe `CollectionGroup` makes more sense here?

Reply:
Yeah good call, let me take another swipe at this. I almost wanna call it `NamedCollectionGroup` since that's what it really is?

Reply:
Are we okay with `collection.subcollections` as the property name on the hub?